src/e2e/java/teammates/e2e/cases/AdminHomePageE2ETest.java

```diff
@@ -26,26 +26,19 @@ public void testAll() {
         AdminHomePage homePage = loginAdminToPage(url, AdminHomePage.class);
 
         ______TS("Test adding instructors with both valid and invalid details");
-
         String name = "AHPUiT Instrúctör WithPlusInEmail";
         String email = "AHPUiT+++_.instr1!@gmail.tmt";
         String institute = "TEAMMATES Test Institute 1";
-
         homePage.queueInstructorForAdding(name, email, institute);
 
         String singleLineDetails = "Instructor With Invalid Email | invalidemail | TEAMMATES Test Institute 1";
-
         homePage.queueInstructorForAdding(singleLineDetails);
 
-        homePage.addAllInstructors();
+        homePage.clickApproveAccountRequestButton(name, email, institute);
 
-        String successMessage = homePage.getMessageForInstructor(0);
+        String successMessage = homePage.getToastTextContent();
         assertTrue(successMessage.contains(
-                "Instructor \"AHPUiT Instrúctör WithPlusInEmail\" has been successfully created"));
-
-        String failureMessage = homePage.getMessageForInstructor(1);
```

Comment:
Try not to remove this negative test case

Reply:
I've added back an error toast for when a single line request has an invalid email under `validateAndAddInstructorDetails`, it works when I add a request manually but when running the e2e test, no toast is shown.
A toast made by the approve account button - under the account request table still works, any ideas why on this is happening?
https://github.com/user-attachments/assets/c985d4dd-5a51-4ca0-90e1-67ee7e48197a
I've tried rerunning the e2e tests after deleting all the postgres data and remaking all docker containers.

Reply:
The issue was user error, I forgot to build the front-end files after editing.
I've added back in the negative test case with a updated error message.